UKHomeOffice/sgmr-service
# SPCR-500 Ensure form data persists on page refresh

### AC / Description of changes
Currently on the arrival, departure and skipper pages of the voyage, when you input data and refresh the page, the data is lost. We want the data to persist when the page is refreshed

### To Test
- Create a voyage
- On the arrival page input some data
- Refresh the page
- > The data should still be visible
- Repeat with the departure and skipper pages
- > All data should persist on page refresh
- From a draft voyage (with data on the departure, arrival and skipper pages)
- On the 3 pages, go to dev tools and delete formData
- Refresh the page
- > The form should refill with the data that was saved previously

### Notes

- Skipper page has been implemented slightly differently to arrival and departure pages so the solution is slightly different
- This is not implemented on the PC pages as this is a more complicated issue

---

**Remember to merge to feature branches not `master`.**
